cmake/CMakeLists.txt

```diff
@@ -241,6 +241,9 @@ option(onnxruntime_ENABLE_CPUINFO "Enable cpuinfo" ON)
 # ATen fallback support
 option(onnxruntime_ENABLE_ATEN "Enable ATen fallback" OFF)
 
+# dlpack support
+option(onnxruntime_ENABLE_DLPACK "Enable dlpack" ON)
```

Comment:
This is on only when building the python package right?

Reply:
Made as you expect: https://github.com/microsoft/onnxruntime/pull/23110/commits/cdbd26da60d7bf6c8e937c6555d96693e4ebac26 
It should work on builds from build.py